source/easyleed/defaultconfig.py

```diff
@@ -1,84 +1,177 @@
-# how much information shall be logged
-import logging
-loggingLevel = logging.INFO
-loggingFilename = "easyleed.log"
-
+'''
+configuration
+------------------
+Class for handling configuration
+'''
+import configparser, logging, os
 import numpy as np
+from pathlib import Path
+from . import __version__
 
-#### IO related ####
-####################
-
-# regular expression to extract energy from filename (not used for img files)
-# match the last (floating-point) number before file extension
-IO_energyRegex = "[0-9]+(\.[0-9]*)?([Ee](|\+|-)[0-9]+)?(?=[^.0-9]*\.[A-Za-z0-9]+$)"
-
-####################
-####################
+class Configuration():
+    def __init__(self):
+        self.home = str(Path.home())+"/"
+        self.configFile = str(self.home+"easyleed.ini")
+        self.loggingFilename = str("easyleed.log")
```

Comment:
No string conversion seems to be needed in these two lines. Or am I missing something?